doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -591,65 +620,20 @@ Induction
 
       This error is given when one side of the equality is not a constructor.
```

Comment:
The first listed error message doesn't exist anymore. It looks like it has been replaced by the following one:

>No information can be deduced from this equality and the injectivity of constructors. This may be because the terms are convertible, or due to pattern matching restrictions in the sort Prop. You can try to use option Set Keep Proof Equalities.

The second listed error message is also not correct. The actual error message is "No primitive equality found." like for `discriminate`.